js/start.js

```diff
@@ -264,15 +266,19 @@ let ownFollowingFailed;
 let exchangeRatesFetch;
 let walletBalanceFetch;
 let walletBalanceFetchFailed;
+let searchProvidersFetch;
+let searchProvidersFetchFailed;
 
 function fetchStartupData() {
   ownFollowingFetch = !ownFollowingFetch || ownFollowingFailed ?
     app.ownFollowing.fetch() : ownFollowingFetch;
   exchangeRatesFetch = exchangeRatesFetch || fetchExchangeRates();
   walletBalanceFetch = !walletBalanceFetch || walletBalanceFetchFailed ?
     app.walletBalance.fetch() : walletBalanceFetch;
+  searchProvidersFetch = !searchProvidersFetch || searchProvidersFetch ?
```

Comment:
This should be:

`searchProvidersFetch = !searchProvidersFetch || searchProvidersFetchFailed ?`